components/servo/Cargo.lock

```diff
@@ -183,6 +183,11 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "bitflags"
+version = "0.4.0"
```

Comment:
You mean the `nix` PR that I filed and mentioned in the comment you just replied to? :)